Index: src/App.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from \"react\";\r\nimport logo from \"./logo.svg\";\r\nimport \"./App.css\";\r\nimport Header from \"./components/Header\";\r\nimport Sidebar from \"./components/Sidebar\";\r\nimport Footer from \"./components/Footer\";\r\n\r\nfunction App() {\r\n  return (\r\n    <div className=\"App\">\r\n      <Header />\r\n      <Sidebar />\r\n      {/* <div>!!!!!!!!!!!!!!!나이키 메인화면 수정중!!!!!!!!!!!!!</div> */}\r\n      <Footer />\r\n    </div>\r\n  );\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.tsx b/src/App.tsx
--- a/src/App.tsx	(revision 94becdea00489cee514e470b90cd8ad426677499)
+++ b/src/App.tsx	(date 1649387004467)
@@ -4,6 +4,7 @@
 import Header from "./components/Header";
 import Sidebar from "./components/Sidebar";
 import Footer from "./components/Footer";
+import Main from "./components/Main";
 
 function App() {
   return (
@@ -11,6 +12,7 @@
       <Header />
       <Sidebar />
       {/* <div>!!!!!!!!!!!!!!!나이키 메인화면 수정중!!!!!!!!!!!!!</div> */}
+      <Main />
       <Footer />
     </div>
   );
Index: src/components/Main.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/Main.tsx b/src/components/Main.tsx
new file mode 100644
--- /dev/null	(date 1649642030585)
+++ b/src/components/Main.tsx	(date 1649642030585)
@@ -0,0 +1,95 @@
+import React from "react";
+import "../css/default.css";
+import "../css/Main.css";
+
+export default function Main() {
+return (
+    <>
+        <div className="content-body">
+            <div className="category">
+                <div className="content-container">
+                    <div>
+                        <div className="nav">
+                            <div className="link-area">
+                                <ul className="first-menu">
+                                    <li className="menu1 active"><a href="">전체 보기</a></li>
+                                    <li className="menu2"><a href="">MEN</a></li>
+                                    <li className="menu3"><a href="">WOMEN</a></li>
+                                    <li className="menu4"><a href="">KIDS</a></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="nr-container">
+                <ul className="item-list-wrap">
+                    <li className="item-list">
+                        <div className="product-item">
+                            <div className="product-img">
+                                <a href="">
+                                    <img src="../images/a2b8a2c3-88b1-4403-9830-8e17005ebe16_primary.jpg" alt=""/>
+                                </a>
+                            </div>
+                        </div>
+                        <div className="product-info">
+                            <div className="product-display">
+                                <div className="product-info-area">
+                                    <p className="product-display-name">나이키 요가</p>
+                                    <div className="product-subtitle">주니어 쇼츠</div>
+                                    <div className="item-color-opt">1 컬러</div>
+                                </div>
+                                <div className="product-price">
+                                    <p className="product-display-price">31,900원</p>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                    <li className="item-list">
+                        <div className="product-item">
+                            <div className="product-img">
+                                <a href="">
+                                    <img src="../images/a2b8a2c3-88b1-4403-9830-8e17005ebe16_primary.jpg" alt=""/>
+                                </a>
+                            </div>
+                        </div>
+                        <div className="product-info">
+                            <div className="product-display">
+                                <div className="product-info-area">
+                                    <p className="product-display-name">나이키 요가</p>
+                                    <div className="product-subtitle">주니어 쇼츠</div>
+                                    <div className="item-color-opt">1 컬러</div>
+                                </div>
+                                <div className="product-price">
+                                    <p className="product-display-price">31,900원</p>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                    <li className="item-list">
+                        <div className="product-item">
+                            <div className="product-img">
+                                <a href="">
+                                    <img src="../images/a2b8a2c3-88b1-4403-9830-8e17005ebe16_primary.jpg" alt=""/>
+                                </a>
+                            </div>
+                        </div>
+                        <div className="product-info">
+                            <div className="product-display">
+                                <div className="product-info-area">
+                                    <p className="product-display-name">나이키 요가</p>
+                                    <div className="product-subtitle">주니어 쇼츠</div>
+                                    <div className="item-color-opt">1 컬러</div>
+                                </div>
+                                <div className="product-price">
+                                    <p className="product-display-price">31,900원</p>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </>
+);
+}
Index: src/css/Main.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/css/Main.css b/src/css/Main.css
new file mode 100644
--- /dev/null	(date 1649642064325)
+++ b/src/css/Main.css	(date 1649642064325)
@@ -0,0 +1,104 @@
+.content-body {
+    min-height: 330px;
+    min-width: 320px;
+    margin-left: 318px;
+    margin-right: 48px;
+    transition: margin-left 0.3s;
+}
+.content-container {
+    overflow: visible;
+}
+.content-container .nav {
+    width: 1537px;
+    top: 0px;
+    position: static;
+    padding: 10px 0;
+    background: #fff;
+    margin: 0 auto;
+}
+.link-area {
+    position: relative;
+    width: 100%;
+    text-align: center;
+    background-color: white;
+}
+.first-menu {
+    height: 100%;
+    margin: 0 auto;
+}
+.first-menu li.active {
+    background: #111;
+    border: 1px solid #111;
+}
+.first-menu li {
+    display: inline-block;
+    height: 100%;
+    margin-right: 10px;
+    padding: 8px 16px;
+    background: #e9e9e9;
+    border-radius: 5px;
+    border: 1px solid #e9e9e9;
+}
+.first-menu li.active a {
+    color: #fff;
+}
+.first-menu li a {
+    color: #787878;
+}
+
+.nr-container {
+    padding-left: 0;
+    padding-right: 0;
+}
+.item-list-wrap {
+    display: flex;
+}
+.item-list-wrap li {
+    width: 33.3%;
+    display: inline-block;
+}
+.item-list {
+    padding-left: 8px;
+    padding-right: 8px;
+}
+.product-item {
+    overflow: hidden;
+}
+.product-img {
+    position: relative;
+    z-index: 2;
+    background: whitesmoke;
+    width: 100%;
+}
+.product-img img {
+    width: 100%;
+    height: 100%;
+    vertical-align: top;
+}
+.product-info {
+    text-align: left;
+    padding-top: 12px;
+}
+.product-info .product-display {
+    display: flex;
+    justify-content: space-between;
+    word-break: keep-all;
+    line-height: 1.6em;
+    letter-spacing: -0.5px;
+}
+.product-info-area {
+    width: 66.6%;
+    padding-right: 15px;
+}
+.product-price {
+    padding-right: 10px;
+}
+.product-display-name {
+    line-height: 1.4em;
+}
+.item-color-opt {
+    padding-top: 3px;
+}
+.product-display-price {
+    white-space: nowrap;
+}
